src/main/resources/templates/shopping-cart.html

```diff
@@ -23,8 +23,7 @@ <h1>Shopping Cart</h1>
     <div th:id="'cart-item-' + ${item.book.id}" th:each="item : ${cart.items}">
             <div class="row">
             <div class="cart-item-img col-xs-height col-xs-4 col-md-4 col-lg-3 col-top">
-                <img src="https://via.placeholder.com/155x200">
-                <!--<img th:attr="src=@{${item.book.picture}}">-->
+                <img class="card-image" th:attr="src=@{${item.book.picture_url}}">
```

Comment:
I think we want a default 'book cover not found' image to also be displayed on this page similar to how it's done in _bookViewer